index.html

```diff
@@ -229,10 +229,56 @@ <h2 class="project-title">Uber Navigation</h2>
               </ul>
             </div>
             <div class="action">
-              <button type="button" class="action-button">See Project</button>
+              <button onclick="a(4)" id="btn-4" type="button" class="action-button">See Project</button>
             </div>
           </div>
         </div>
+        <!-- Card-aux -->
+      <div class="container-card-aux">
+        <div class="card-aux">
+          <div class="header-aux">
+            <div class="title-aux">
+              <h2 id="title-a">Aux</h2>
```

Comment:
- Kindly your mobile popup is not scrollable so it will be cut to `375`, stick to the Figma design and make it scrollable for better user interactivity 👍👍

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/217464369-f238ff43-8a35-45ee-906b-b1c61214b67b.png" width="300">|<img src="https://user-images.githubusercontent.com/52010789/217464358-ad8d3f48-25e7-4c3d-8cc4-3494f1f3bad5.png" width="300">|


😎️

🤝️

Reply:
> Correction

I have corrected the styling based on the figma template.

I have corrected the offset by adding the `overfloat` property.